js/main.js

```diff
@@ -0,0 +1,90 @@
+
+var userTitle = document.querySelector('.user-title');
+var form = document.querySelector('form');
+var resultUL = document.querySelector('.result-box');
+var views = document.querySelectorAll('.view');
+var home = document.querySelector('.home');
+
+home.addEventListener('click', function () {
+  viewSwitch('search-form');
+});
+
+function saveInfo(event) {
+  event.preventDefault();
+  while (resultUL.firstElementChild) {
+    resultUL.firstElementChild.remove();
+  }
+  data.tempTitle = userTitle.value;
+  searchRequest(data.tempTitle);
+  viewSwitch('search-result');
+}
+
+form.addEventListener('submit', saveInfo);
+
+function createSearch(result) {
+  var list = document.createElement('li');
+  list.className = 'search-list-item';
+  var row = document.createElement('div');
+  row.className = 'row';
+  list.appendChild(row);
+  var col = document.createElement('div');
+  col.className = 'col-full';
+  row.appendChild(col);
+  var theSearch = document.createElement('div');
+  theSearch.className = 'a-search';
+  col.appendChild(theSearch);
+  var poster = document.createElement('img');
+  poster.className = 'search-image';
+  poster.setAttribute('src', result.poster);
+  theSearch.appendChild(poster);
+  var centerDiv = document.createElement('div');
+  centerDiv.className = 'center';
+  theSearch.appendChild(centerDiv);
+  var name = document.createElement('p');
+  name.className = 'search-title';
+  name.textContent = result.title;
+  centerDiv.appendChild(name);
+  return list;
+}
+
+function domLoad(event) {
+  while (resultUL.firstElementChild) {
+    resultUL.firstElementChild.remove();
+  }
+  for (var i = 0; i < data.search.length; i++) {
+    resultUL.append(createSearch(data.search[i]));
+  }
```

Comment:
What is the purpose of this? Is this to preserve the results of the previous search from the previous session of the app being used? I don't think this is necessary, you should only show results for a search after it has been made in real-time. Google doesn't open to the results of your previous search on a previous session, for example.